webapi/webapi.go

```diff
@@ -255,7 +255,7 @@ func router(debugMode bool, cookieSecret []byte, dcrd rpc.DcrdConnect, wallets r
 	basic := router.Group("/admin").Use(
 		withDcrdClient(dcrd), withWalletClients(wallets), gin.BasicAuth(gin.Accounts{
 			"admin": cfg.AdminPass,
-		}),
+		}), 
```

Comment:
Seems you added a whitespace here that shouldn't be. gofmt should correct all formatting errors anyways, including this.